packages/client/src/Dashboard.tsx

```diff
@@ -120,14 +121,23 @@ const useStyles = makeStyles(theme => ({
 export default function Dashboard({ accountId, navigate }: Props) {
   const classes = useStyles()
   const [open, setOpen] = React.useState(false)
-  const [isSearching, setIsSearching] = React.useState(false)
-  const [searchQuery, setSearchQuery] = React.useState("")
+  // const [isSearching, setIsSearching] = React.useState(false)
```

Comment:
Please remove commented-out code. We can always look back at revision history if we need to see what was there.